### PR TITLE
Initialize behat

### PR DESCRIPTION
**Pull request by BenoitLeveque**
- Add behat dependencies to composer
- create a basic behat.yml file

### CODE REVIEW

File: Command/InitializeBehatCommand.php [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+use Symfony\Bridge\Twig\TwigEngine;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\DialogHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpKernel\Kernel;
+use Symfony\Component\Process\Process;
+
+class InitializeBehatCommand extends ContainerAwareCommand
```
Comment (nicolas-bastien) [inline Command/InitializeBehatCommand.php]: should extend AbstractInitializerCommand

File: Command/InitializeBehatCommand.php [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+        if (!$process->isSuccessful()) {
+            throw new RuntimeException($process->getErrorOutput());
+        }
+    }
+
+    /**
+     * @return DialogHelper
+     */
+    private function getDialog()
```
Comment (nicolas-bastien) [inline Command/InitializeBehatCommand.php]: Can you refator this utils method in the abstract command ?

File: Command/InitializeBehatCommand.php (hunk 1 above)
Comment (BenoitLeveque) [inline Command/InitializeBehatCommand.php]: this class is useless for my command.

**Comment by BenoitLeveque** [comment]
@nicolas-bastien this PR can be merged

**Pull request merged**